src/app/[locale]/specs/sync/page.mdx

```diff
@@ -0,0 +1,171 @@
+
+Event Fields:
+
+- `seq` (integer, required): same for all event types
+- `did` (string with DID syntax, required): same for all event types
+- `time` (string with datetime syntax, required): same for all event types
+- `active` (boolean, required): whether the repository is currently available and can be redistributed
+- `status` (string, optional): string status code which describes the account state in more detail. Known values include:
+    - `takendown`: indefinite removal of the repository by a service provider, due to a terms or policy violation
+    - `suspended`: temporary or time-limited variant of `takedown`
+    - `deleted`: account has been deactivated, possibly permanently.
+    - `deactivated`: temporary or indefinite removal of all public data by the account themselves.
+
+When coming from any service which redistributes account data, the event describes what the new status is *at that service*, and is authoritative in that context. In other words, the event is hop-by-hop for repository hosts and mirrors.
+
+See the Account Hosting specification for more details.
+
+### `#commit` Events
+
+This event indicates that there has been a new repository commit for the indicated account. The event usually contains the "diff" of repository data, in the form of a CAR slice.
+
+See the Repository specification for more details around repo diffs.
+
+Event Fields:
+
+- `seq` (integer, required): same for all event types
+- `repo` (string with DID syntax, required): the same as `did` for all other event types
+- `time` (string with datetime syntax, required): same for all event types
+- `rev` (string with TID syntax, required): the revision of the commit. Must match the `rev` in the commit block itself.
+- `since` (string with TID syntax, nullable): indicates the `rev` of a preceding commit, which the the repo diff contains differences from
+- `commit` (cid-link, required): CID of the commit object (in `blocks`)
+- `tooBig` (boolean, required): if true, indicates that the repo diff was too large, and that `blocks`, `ops`, and complete `blobs` are not all included
+- `blocks` (bytes, required): CAR "slice" for the corresponding repo diff. The commit object must always be included.
+- `ops` (array of objects, required): list of record-level operations in this commit: specific records created, updated, deleted
+- `blobs` (array of cid-link, required): set of new blobs (by CID) referenced by records in this commit
+
+Commit events are broadcast when the account repository changes. Commits can be "empty", meaning no actual record content changed, and only the `rev` was incremented. They can contain a single record update, or multiple updates. Only the commit object, record blocks, and MST tree nodes are authenticated (signed): the `since`, `ops`, `blobs`, and `tooBig` flag are not self-certifying, and could in theory be manipulated, or otherwise be incorrect or incomplete.
```

Comment:
```suggestion
Commit events are broadcast when the account repository changes. Commits can be "empty", meaning no actual record content changed, and only the `rev` was incremented. They can contain a single record update, or multiple updates. Only the commit object, record blocks, and MST tree nodes are authenticated (i.e., signed): the `since`, `ops`, `blobs`, and `tooBig` properties are not self-certifying, and could in theory be manipulated, or otherwise be incorrect or incomplete.
```